libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2240,6 +2243,16 @@ Expression const* SMTEncoder::leftmostBase(IndexAccess const& _indexAccess)
 	return base;
 }
 
+TypePointer SMTEncoder::keyType(TypePointer _type)
+{
+	if (auto const* mapType = dynamic_cast<MappingType const*>(_type))
+		return mapType->keyType();
```

Comment:
```suggestion
	if (auto const* mappingType = dynamic_cast<MappingType const*>(_type))
		return mappingType->keyType();
```
Minor comment. `mappingType` is more consistently used in the codebase.

Reply:
Updated